src/pages/Friends/request.js

```diff
@@ -20,163 +20,149 @@ import TableHead from "@mui/material/TableHead";
 import TableRow from "@mui/material/TableRow";
 import Paper from "@mui/material/Paper";
 import Button from "@mui/material/Button";
-import Sidebar from '../../components/Sidebar/sidebar';
-
-import AppBar from '@mui/material/AppBar';
-import Box from '@mui/material/Box';
-import Toolbar from '@mui/material/Toolbar';
-import Typography from '@mui/material/Typography';
-
+import Sidebar from "../../components/Sidebar/sidebar";
 
+import AppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import Toolbar from "@mui/material/Toolbar";
+import Typography from "@mui/material/Typography";
 
 function Request() {
-
   const user = useSelector((state) => state.user);
-  const [follow_list, setList] = useState({"items": []}); 
-  const [success, setSuccess] = useState(null); 
+  const [follow_list, setList] = useState({ items: [] });
+  const [success, setSuccess] = useState(null);
   const navigate = useNavigate();
-  
 
-  useEffect(() => { 
-      get_request(user.id, setList)
-    
+  useEffect(() => {
+    get_request(user.id, setList);
   }, []);
 
-  const DeleteRequest= (actor_id) => {
-      delete_request(user.id, actor_id, onSuccess)
-  }
+  const DeleteRequest = (actor_id) => {
+    delete_request(user.id, actor_id, onSuccess);
+  };
 
-  const AcceptRequest= (actor_id) => {
-      add_followers_for_author(user.id, actor_id, onSuccess)
-      delete_request(user.id, actor_id, onSuccess)
-  }
+  const AcceptRequest = (actor_id) => {
+    add_followers_for_author(user.id, actor_id, onSuccess);
+    delete_request(user.id, actor_id, onSuccess);
+  };
 
   const onSuccess = () => {
-      setSuccess(true);
-  }
-  
-  
+    setSuccess(true);
+  };
+
   const goBack = () => {
-      navigate("/");
+    navigate("/");
   };
 
   let page = 1;
   const page_buttons = () => {
-
-    if (follow_list.items.length < 5 && page === 1)
-    {
+    if (follow_list.items.length < 5 && page === 1) {
       return;
     }
-    if (page === 1)
-    {
-        return (<button onClick={forward_page}>Next Page</button>);
-    } 
-    else if (follow_list.items.length < 5)
-    {
-      return <button onClick={back_page}>Prev Page</button>
-    } 
-    else 
-    {
+    if (page === 1) {
+      return <button onClick={forward_page}>Next Page</button>;
+    } else if (follow_list.items.length < 5) {
```

Comment:
And here

Reply:
I just updated get_all_authors api to get 10 authors so leave other as is for now but yeah I can do that for other calls